base/db/DESCRIPTION

```diff
@@ -25,7 +25,12 @@ Imports:
     udunits2
 Suggests:
     RPostgreSQL,
-    testthat (>= 1.0.2)
+    RSQLite,
+    testthat (>= 1.0.2),
+    tidyverse,
+    data.table,
+    rcrossref,
+    here
```

Comment:
Good catch on the `R CMD check`. I'll drop the `.here` file, since `here` should be able to find the package root anyway.

In general, I'm including `here` only as a Suggest, and it's never actually used in any package functions (at least, I'm pretty sure it isn't; I'll have to check) -- only in the TRY import scripts, which are meant to be run interactively. I'm aware of `rprojroot`, but I didn't think the complexity was worth it for these scripts.